docker-compose.dev.yml

```diff
@@ -12,6 +12,7 @@ x-sbt-volumes:
     - node_modules-data:/usr/src/server/node_modules
     - project-data:/usr/src/server/project/project
     - project-target-data:/usr/src/server/project/target
+    - project-coverage:/usr/src/server/target/scala-2.13/jacoco/report
```

Comment:
Remove project-coverage

Reply:
@kmcnellis is the rationale that we'd want to mirror the local filesystem for Jacoco reports rather than putting in on a shared Docker volume?

If that's the rationale, I think I'd agree. That way one could just have the coverage report show up in their local filesystem after a run rather than needing to start a Docker container with the volume attached in order to view it.

Reply:
Yes, to both of you.  I think this was just an artifact from the first approach